tool/e2e-tests/helpers.ts

```diff
@@ -164,6 +165,43 @@ export async function claimAmendPrescriptionUserJourney(
   await checkApiResult(driver)
 }
 
+export async function editPrescriptionOrganisationUserJourney(
+  driver: ThenableWebDriver,
+  newOrganisation: string
+): Promise<string> {
+  await loginViaSimulatedAuthSmartcardUser(driver)
+  await setMockSigningConfig(driver)
+  await createPrescription(driver)
+  await loadPredefinedExamplePrescription(driver)
+  await editPrescriptionOrganisation(driver, newOrganisation)
+  await sendPrescription(driver)
+  await checkApiResult(driver)
+  return await getCreatedPrescriptionId(driver)
+}
+
+export async function editPrescriptionOrganisation(
```

Comment:
Possibly worth tidying up where the test functions are, in the test file if they're not shared, in helpers if they are?

Reply:
Moved across